package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
+}
```

Comment:
Remove this file - we have things set up in two different packages, one for client and one for server. Each will have its respective package.json files, so there will be no top-level package-lock.json file